sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_statsbeat.py

```diff
@@ -1,77 +1,17 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-import threading
-
-from opentelemetry.sdk.metrics import MeterProvider
-from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader
-from opentelemetry.sdk.resources import Resource
-
-from azure.monitor.opentelemetry.exporter.statsbeat._exporter import _StatsBeatExporter
-from azure.monitor.opentelemetry.exporter.statsbeat._statsbeat_metrics import _StatsbeatMetrics
-from azure.monitor.opentelemetry.exporter.statsbeat._state import (
-    _STATSBEAT_STATE,
-    _STATSBEAT_STATE_LOCK,
-)
-from azure.monitor.opentelemetry.exporter.statsbeat._utils import (
-    _get_stats_connection_string,
-    _get_stats_long_export_interval,
-    _get_stats_short_export_interval,
+from azure.monitor.opentelemetry.exporter.statsbeat._manager import (
+    StatsbeatConfig,
+    StatsbeatManager,
 )
 
 
-_STATSBEAT_METRICS = None
-_STATSBEAT_LOCK = threading.Lock()
-
-
-# pylint: disable=global-statement
-# pylint: disable=protected-access
 def collect_statsbeat_metrics(exporter) -> None:
-    global _STATSBEAT_METRICS
```

Comment:
We remove all this logic into `_manager.py`, We leave the functions here for now to be backwards compatible.